src/github.com/stellar/horizon/actions_payment.go

```diff
@@ -48,10 +53,17 @@ func (action *PaymentsIndexAction) SSE(stream sse.Stream) {
 			records := action.Records[stream.SentCount():]
 
 			for _, record := range records {
-				res, err := resource.NewOperation(action.Ctx, record)
+				ledger, found := action.Ledgers.Records[record.LedgerSequence()]
```

Comment:
Shouldn't it also call `action.loadLedgers` before this function?

Reply:
Good catch!